src/components/Utils/SheskaCardDef.tsx

```diff
@@ -2,35 +2,90 @@ import React from "react";
 import MiniCard from "../SheskaList/MiniCard";
 
 /**
- * This class is used to represent a SheskaCard. It is used to store the data for a SheskaCard and to generate the JSX for a SheskaCard.
- * @class SheskaCard - This class is used to represent a SheskaCard. It is used to store the data for a SheskaCard and to generate the JSX for a SheskaCard.
- * @property {string} _cardID - The ID of the SheskaCard
- * @property {string} _title - The title of the SheskaCard
- * @property {string} _subtitle - The subtitle of the SheskaCard
- * @property {string} _description - The description of the SheskaCard
- * @property {string[]} _imageURLs - The image URLs of the SheskaCard
+ * SheskaCard is a class that defines a card object.
+ * It contains all the information needed to display a card on the SheskaList page
+ * It also contains methods to display the card in different ways
+ * @param cardID the id of the card
+ * @param title the title of the card
+ * @param subtitle the subtitle of the card
+ * @param description the description of the card
+ * @param imageOrder the order of the images in the card
+ * @param expectedAverage the expected average of the card default is [0,0]
+ * @param goal the goal of the card default is [0,0]
+ * @param amountRaised the amount raised of the card default is [0,0]
+ * @param guestsAbsorbFees whether or not guests absorb fees default is [0,0]
+ * @param dateCreated the date the card was created
+ * @param dateUpdated the date the card was last updated
  */
 class SheskaCard {
     private _cardID: string;
     private _title: string;
     private _subtitle: string;
     private _description?: string;
     private _imageOrder?: string[];
+    private _amountRequested?: any;
+    private _amountRaised?: any;
+    private _expectedAverage?: any;
+    private _guestsAbsorbFees?: boolean;
+    private _dateCreated?: string;
+    private _dateUpdated?: string;
 
-    constructor(cardID: string, title: string, subtitle: string, description?: string, imageOrder?: string[],) {
+
+
+    constructor(cardID: string, title: string, subtitle: string, description?: string, imageOrder?: string[], expectedAverage? : string[],
+                goal?: string[], amountRaised?: string[], guestsAbsorbFees?: boolean, dateCreated?: string, dateUpdated?: string) {
         this._cardID = cardID;
         this._title = title;
         this._subtitle = subtitle;
         if(description) {
             this._description = description;
+        } else {
+            this._description = '';
```

Comment:
`this._description = description || '';`

Reply:
use undefined instead of ''?

Reply:
fixed